polling_stations/apps/addressbase/management/commands/create_uprn_council_lookup.py

```diff
@@ -48,7 +48,8 @@ def handle(self, *args, **kwargs):
             COPY (SELECT
                     a.uprn as uprn,
                     c.gss as lad,
-                    '' as polling_station_id
+                    '' as polling_station_id,
+                    null as advance_voting_station_id
```

Comment:
When you re-import this from the CSV, will it actually import it as a `NULL`? (I've not tested it)

Reply:
Yep!